build/Dockerfile

```diff
@@ -54,6 +56,54 @@ RUN ln -s ${MACHINE_SNAPSHOT_PATH}/0_0 ${MACHINE_SNAPSHOT_PATH}/latest
 WORKDIR ${MACHINE_SNAPSHOT_PATH}
 CMD [ "/bin/bash" ]
 
+#
+# Devnet generation stage
+#
+FROM debian:bookworm-slim as rollups-devnet
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    git \
+    jq \
+    wget \
+    xxd
+
+# Install foundry
+RUN curl -L https://foundry.paradigm.xyz | bash
+SHELL ["/bin/bash", "-c"]
```

Comment:
Why do we need to change the shell?

Reply:
It's a workaround for https://github.com/foundry-rs/foundry/issues/2476. Another alternative would be calling `foundryup` like this:
```docker
RUN ~/.foundry/bin/foundryup
```